GLTFSDK/Source/Validation.cpp

```diff
@@ -88,10 +88,33 @@ namespace
         Validation::ValidateBufferView(bufferView, buffer);
     }
 
-    void ValidateTriangleVertexCount(const MeshMode mode, const size_t count, const std::string& type)
+    void ValidateVertexCount(const MeshMode mode, const size_t count, const std::string& type)
     {
         switch (mode)
         {
+        case MESH_POINTS:
+            break;
+
+        case MESH_LINES:
+            if (count < 2)
+            {
+                throw ValidationException(type + " count must be at least 2.");
```

Comment:
Are there any tests that test the failure cases?  It looks like all the test files are confirming success, but that only seems to test that you added "case MESH_XXX" to the switch statement, but not much around whether the failure logic in them is correct.

Reply:
Consider, for example, how someone could change this code so that all of the unit tests still pass while also not working correctly. It would be quite easy to break this code without breaking any of the tests.